packages/web/src/schema/song-schema.js

```diff
@@ -0,0 +1,7 @@
+import { normalize, schema } from "normalizr";
+
+export const song = new schema.Entity("songs", {}, { idAttribute: "_id" });
```

Comment:
This code is repeated in [`playlist-schema.js`](https://github.com/ANFAIRA/apollofy-anfaira/pull/127/files#diff-837f8ee7d8350f29c4c09892d22cd79c568a131d59b7eff2054340ead9f571d1R5). It may be useful to define all schemas in one file or to export schemas that are used elsewhere, to DRY it.